packages/react/src/isomorphicClerk.ts

```diff
@@ -87,8 +87,9 @@ export default class IsomorphicClerk {
   static getOrCreateInstance(options: IsomorphicClerkOptions) {
     // During SSR: a new instance should be created for every request
     // During CSR: use the cached instance for the whole lifetime of the app
+    // Also will recreate the instance if the provided Clerk instance changes
     // This method should be idempotent in both scenarios
-    if (!inClientSide() || !this.#instance) {
+    if (!inClientSide() || !this.#instance || (options.Clerk && this.#instance.Clerk !== options.Clerk)) {
```

Comment:
Without this change, there was no way to forcibly reset the created `IsomorphicClerk` instance. Given that we have not considered the possibility of changing the instance of `Clerk` at runtime, adding this additional check seems reasonable given the new requirement.

Reply:
This makes sense 👍🏻